pkg/gameservers/migration_test.go

```diff
@@ -137,6 +137,7 @@ func TestMigrationControllerSyncGameServer(t *testing.T) {
 					assert.Equal(t, ipChangeFixture, gs.Status.Address)
 				},
 				postTests: func(t *testing.T, m agtesting.Mocks) {
+					agtesting.AssertEventContains(t, m.FakeRecorder.Events, "Warning Scheduled GameServer/Node address mismatch: gs.Name=test: gs.Status.Address=12.12.12.12, node.Status.Addresses=99.99.99.99")
```

Comment:
Does this mean we get two events on a node migration?

Do we need an extra event, or can it all be combined?

Reply:
I think I would just drop this to a log message (which is I think what you had originally, but I might be mis-remembering).